Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/logout/LogoutPage.module.scss

```diff
@@ -0,0 +1,8 @@
+﻿@use 'src/styles/fonts';
+
+.logout {
+  display: flex;
+  font: fonts.$font-family-lato;
```

Comment:
Order flex properties meaning justify-content: center; to be just under display flex. This way its easier to read

Reply:
If I try to do that, then there is an error with ESLint due to the properties not being in alphabetical order.

Reply:
Okay